### PR TITLE
feat: add min votes required for imdb rating to be added

### PR DESCRIPTION
**Pull request by Primdahl26**
# Description

This makes us able to control the min amount of votes media needs to get a imdb rating
Fixes #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: backend/cli.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
 
 @app.command()
-def add_imdb_ratings():
+def add_imdb_ratings(min_votes: int = 100):
```
Comment (AndreasPB) [inline backend/cli.py:38]: I'm not sure if we'll miss a lot with a minimum of 100 - Maybe 25 is enough

File: backend/cli.py [hunk 2]
```diff
@@ -71,8 +71,12 @@ def add_imdb_ratings():
     typer.echo("Adding ratings to meilisearch...")
 
     imdb_media = [
-        {"id": media.id, "imdb_rating": float(imdb_ratings[media.imdb_id])}
+        {
+            "id": media.id,
+            "imdb_rating": float(imdb_ratings[media.imdb_id].get("rating")),
+        }
         if imdb_ratings.get(media.imdb_id)
+        and int(imdb_ratings[media.imdb_id].get("votes")) >= min_votes
```
Comment (AndreasPB) [inline backend/cli.py:79]: Is there a reason we are mixing `[]`'s and `get()`'s?

File: backend/cli.py (hunk 1 above)
Comment (Primdahl26) [reply backend/cli.py]: i did 50 :D

File: backend/cli.py (hunk 2 above)
Comment (Primdahl26) [reply backend/cli.py]: not really no - changed it

**Review by AndreasPB — APPROVED** [review]

**Pull request merged**